modules/data.atmosphere/R/met2CF.csv.R

```diff
@@ -158,22 +170,19 @@ met2CF.csv <- function(in.path, in.prefix, outfolder, start_date, end_date, form
                                           alldat[, format$vars$input_name[DOY_index]]), format = "%Y-%j"), format = "%Y-%m-%d")
         hh          <- floor(alldat[, format$vars$input_name[hour_index]])
         mm          <- (alldat[, format$vars$input_name[hour_index]] - hh) * 60
-        yyddhhmm    <- strptime(paste0(yearday, " ", hh, ":", mm), format = "%Y-%m-%d %H:%M")
+        yyddhhmm    <- strptime(paste0(yearday, " ", hh, ":", mm), format = "%Y-%m-%d %H:%M", tz=tz)
         alldatetime <- as.POSIXct(yyddhhmm)
       } else {
         ## Does not match any of the known date formats, add new ones here!
         PEcAn.utils::logger.error("datetime column is not specified in format")
       }
     } else {
-      datetime_units <- format$vars$input_units[datetime_index]  #lubridate function to call such as ymd_hms
+      datetime_raw <- alldat[, format$vars$input_name[datetime_index]]
+      datetime_units <- paste(format$vars$storage_type[datetime_index],collapse = " ")  #strptime convention
       if (datetime_units == "") {
-        datetime_units <- "ymd_hm"
+        datetime_units <- "%Y%m%d%H%M" #assume ISO convention
```

Comment:
This is the default if no string is specified in format record storage_type. This works for Fluxnet and ameriflux (tested through web interface) which use dates like 200401011330.